extensions/raycast-quick-search/src/deepl-trans.ts

```diff
@@ -0,0 +1,17 @@
+import { showHUD } from "@raycast/api";
+import { DeeplOpner } from "./utils/opener";
+import { isNotEmpty, readtext } from "./utils/readtxt";
+
+export default async () => {
+  try {
+    const text = await readtext();
+    if (isNotEmpty(text)) {
+      await DeeplOpner(text);
+      showHUD("🎉 Open Deepl search");
+    } else {
+      showHUD("❌ No text found in clipboard");
+    }
+  } catch (error) {
+    showHUD("💩 Sorry, Can not open Deepl for now!");
```

Comment:
```suggestion
    showHUD("❌ Cannot open Deepl!");
```

Reply:
I'd suggest to keep it brief and maybe you can check whether there's a meaningful short error message that you could extract and append?

Reply:
Same suggestion applies to all other search cases.